packages/manager/apps/cloud/client/app/vrack/add/vrack-add.html

```diff
@@ -28,9 +28,9 @@ <h2 class="text-center" data-translate="vrack_validationPending" data-translate-
                 <div data-ng-if="!VrackAddCtrl.loaders.validationPending">
                     <!-- contracts -->
                     <div class="row" data-ng-if="!VrackAddCtrl.model.contractsAccepted && VrackAddCtrl.model.agreements.length"
-                         data-contracts="VrackAddCtrl.model.agreements"
+                         data-cuc-contracts="VrackAddCtrl.model.agreements"
```

Comment:
You will have conflict with #1742

@ganeshkumar1989 Are there any requirements to prefer using the directive from `ovhContract` instead of this one?

Reply:
I migrate it as it works before. Refactor is not the goal of this PR.

Reply:
Meaning that #1742 depends on the `cucContracts` directive.

Thanks!